nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-parameter-contexts.js

```diff
@@ -0,0 +1,2298 @@
+                    buttonText: 'Cancel',
+                    color: {
+                        base: '#E3E8EB',
+                        hover: '#C7D2D7',
+                        text: '#004849'
+                    },
+                    handler: {
+                        click: function () {
+                            deferred.resolve();
+
+                            if ($('#parameter-referencing-components-container').is(':visible')) {
+                                updateReferencingComponentsBorder($('#parameter-referencing-components-container'));
+                            }
+
+                            close();
+                        }
+                    }
+                }]);
+            };
+
+            var cancelled = false;
+
+            // update the button model to show the cancel button
+            $('#parameter-context-dialog').modal('setButtonModel', [{
+                buttonText: 'Cancel',
+                color: {
+                    base: '#E3E8EB',
+                    hover: '#C7D2D7',
+                    text: '#004849'
+                },
+                handler: {
+                    click: function () {
+                        cancelled = true;
+
+                        if ($('#parameter-referencing-components-container').is(':visible')) {
+                            updateReferencingComponentsBorder($('#parameter-referencing-components-container'));
+                        }
+
+                        updateToCloseButtonModel();
+                    }
+                }
+            }]);
+
+            var requestId;
+            var handleAjaxFailure = function (xhr, status, error) {
+                // delete the request if possible
+                if (nfCommon.isDefinedAndNotNull(requestId)) {
+                    deleteUpdateRequest(parameterContextEntity.id, requestId);
+                }
+
+                // update the step status
+                $('#parameter-context-update-steps').find('div.parameter-context-step.ajax-loading').removeClass('ajax-loading').addClass('ajax-error');
+
+                if ($('#parameter-referencing-components-container').is(':visible')) {
+                    updateReferencingComponentsBorder($('#parameter-referencing-components-container'));
+                }
+
+                // update the button model
+                updateToApplyOrCancelButtonModel();
+            };
+
+            submitUpdateRequest(parameterContextEntity).done(function (response) {
+                var pollUpdateRequest = function (updateRequestEntity) {
+                    var updateRequest = updateRequestEntity.request;
+                    var errored = nfCommon.isDefinedAndNotNull(updateRequest.failureReason);
+
+                    // get the request id
+                    requestId = updateRequest.requestId;
+
+                    // update the affected components
+                    populateAffectedComponents(updateRequest.affectedComponents);
```

Comment:
At this point (at least for my flow) when updating a parameter context that has a parameter that is referenced by a processor, the server is responding with an empty array for `affectedComponents`... this then clears out the referencing components panel in the Update Parameter Context dialog. So it appears that there were no affected/referencing components

Reply:
Update on this one. This seems to only happen when you are only updating the description of a parameter and not the value. Maybe this is as designed since changing the descriptions probably doesn't require any changes to the flow.